src/lib.rs

```diff
@@ -25,50 +27,36 @@ leptos_i18n::load_locales!();
 pub fn App() -> impl IntoView {
   provide_meta_context();
   provide_i18n_context();
-
-  let authenticated = create_rw_signal::<bool>(false);
-  provide_context(authenticated);
+  provide_query_client();
 
   let (is_routing, set_is_routing) = create_signal(false);
 
   view! {
-    <Stylesheet id="leptos" href="/pkg/lemmy-ui-leptos.css"/>
-    <Link rel="shortcut icon" type_="image/ico" href="/favicon.ico"/>
-    <Meta name="description" content="Lemmy-UI-Leptos."/>
-    <Meta name="viewport" content="viewport-fit=cover"/>
-    <Script src="//cdn.jsdelivr.net/npm/eruda"/>
-    <Script>eruda.init();</Script>
-    <Title text="Brand from env"/>
-
-    // adding `set_is_routing` causes the router to wait for async data to load on new pages
     <Router set_is_routing>
-      <div class="flex flex-col h-screen">
-        <RoutingProgress is_routing max_time=std::time::Duration::from_millis(250)/>
-        <TopNav/>
-        <Routes>
-          <Route path="" view=HomeActivity ssr=SsrMode::Async/>
-          <Route path="home" view=HomeActivity ssr=SsrMode::Async/>
+      <Routes>
+        <Route path="/" view=move || view! { <Layout is_routing/> } ssr=SsrMode::PartiallyBlocked>
```

Comment:
this is cool! i think `SsrMode::PartiallyBlocked` . i think this might be new since i last looked

seems like the perfect balance for SSR performance. looking forward to see how this works.